lib/post_message_definition.yml

```diff
@@ -16,6 +16,10 @@ post_messages:
       ping:
         payload:
           <<: *user_session_properties
+      go_back:
```

Comment:
We use camel case for all other post messages, let's do the same here. Also, let's def add a warning to this entry like we do in `oauthRequested`. On the warning for this message we can straight up not tell users to pass their own callback. Or better yet, we can update the generators so they don't include this in the docs at all.

Reply:
Ah good catch on the case. 

In regards to the warning message, I'm a little confused. Can you explain further what you'd like it to contain?